doc/sphinx/addendum/sprop.rst

```diff
@@ -241,15 +241,33 @@ The term :g:`c (top -> top) (fun x => x) c` infinitely reduces to itself.
 Lack of tactic support
```

Comment:
Is this still a good title for the section?  Maybe something more like "Debugging |SProp| relevance bugs"?